### PR TITLE
Pods 3866

### PR DESCRIPTION
**Pull request by vinnicombe**

### CODE REVIEW

File: app/forms/chargeA/ChargeDetailsFormProvider.scala [hunk 1]
```diff
@@ -21,8 +21,15 @@ import javax.inject.Inject
 import models.chargeA.ChargeDetails
 import play.api.data.Form
 import play.api.data.Forms.mapping
+import uk.gov.voa.play.form.Condition
+import uk.gov.voa.play.form.ConditionalMappings._
 
 class ChargeDetailsFormProvider @Inject() extends Mappings with Constraints {
+  private def condition(field: String): Condition =
```
Comment (grantrobson) [inline app/forms/chargeA/ChargeDetailsFormProvider.scala]: Maybe call this something more meaningful, e.g. isEmptyOrBothHaveValue

File: app/forms/chargeD/ChargeDetailsFormProvider.scala [hunk 2]
```diff
@@ -16,14 +16,21 @@
...
 import models.chargeD.ChargeDDetails
 import play.api.data.Form
 import play.api.data.Forms.mapping
 import play.api.i18n.Messages
+import uk.gov.voa.play.form.Condition
+import uk.gov.voa.play.form.ConditionalMappings._
 
-class ChargeDetailsFormProvider @Inject() extends Mappings with Constraints {
+class ChargeDetailsFormProvider @Inject() extends Mappings with Constraints with Formatters {
+  private def condition(field: String): Condition =
```
Comment (grantrobson) [inline app/forms/chargeD/ChargeDetailsFormProvider.scala]: See previous comment

File: app/forms/chargeA/ChargeDetailsFormProvider.scala (hunk 1 above)
Comment (vinnicombe) [reply app/forms/chargeA/ChargeDetailsFormProvider.scala]: Done

File: app/forms/chargeD/ChargeDetailsFormProvider.scala (hunk 2 above)
Comment (vinnicombe) [reply app/forms/chargeD/ChargeDetailsFormProvider.scala]: Done

**Pull request merged**